firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1846,6 +1855,14 @@ class BrowserViewController: UIViewController,
             navigationHandler?.showEnhancedTrackingProtection(sourceView: view)
         case .menu:
         	didTapOnMenu(button: state.buttonTapped)
+        case .tabTray:
+            focusOnTabSegment()
+            TelemetryWrapper.recordEvent(
```

Comment:
indent type nit

Reply:
I think this might is Github showing this weirdly. Looks good in Xcode. Also tried re-indenting but it ends up being the same.